Database/CleanArchive.py

```diff
@@ -88,12 +96,69 @@ def clean(all=True):
         transaction = {
             '$pull': {'snippets': {'transcript': ''}}
         }
+        if duplicate < 1.0:
+            print('Duplicate found')
+            set_fields['duplicate'] = True
+            duplicates['metadata.Subtitle'] += 1
+        else:
+            set_fields['duplicate'] = False
```

Comment:
If not a duplicate, just ignore the field. We can search using the $exists method.